### PR TITLE
Harold Farmer and Stan Hostetler Changes

### PR DESCRIPTION
**Pull request by haroldfarmer**
User functionality improved by implementation of GUI. Will continue to work on improvements in future projects.

### CODE REVIEW

**Comment by mikeknows** [comment]
Merge conflict with the imports after merging the logging class. Please fix imports and I will merge.

Thanks

**Comment by haroldfarmer** [comment]
Fixed the imports

File: src/wrestlingtournamentcli/Main.java [hunk 1]
```diff
@@ -1,139 +1,355 @@
...
-                + "HELP //Display list of available commands\n"
-                + "LOAD TournamentName //Loads all files associated with this tournament name\n"
-                + "SAVE TournamentName //Saves all information associated with this tournament to files with tournamentName\n"
-                + "NAME TournamentName //Changes the tournament's name"
-                + "IMPORT-TEAMS FileName //Parses the provided file for Team objects\n"
-                + "IMPORT-WRESTLERS FileName //Parses the provided file for Wrestler objects\n"
-                + "VIEW-WRESTLER WrestlerName //Looks for the wrestler and prints his/her information\n"
-                + "UPDATE-MATCH matchNumber winningColor greenPoints redPoints fallType(int) fallTime\n");
-                }
-}
```
Comment (mikeknows) [inline src/wrestlingtournamentcli/Main.java:139]: Need to be able to see the changes made.

File: .classpath [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/">
```
Comment (mikeknows) [inline .classpath:4]: DIscard or explain changes to .classpath

**Review by mikeknows — COMMENTED** [review]
What are the filehashes.bin and filehases.lock files for?

**Comment by haroldfarmer** [comment]
Do you still have office hours or a discord? Im a little confused on why .classpath is changed and why filehashes.bin and filehases.lock is showing up. We didn't touch or create any of these files. I also am a little confused on how git works.

**Comment by mikeknows** [comment]
The computer science dept. has made a discord server with 3120 https://discord.gg/ks3pV7G

Also there is a slack server: raiderplanner.slack.com

I'll be available in both.

**Comment by haroldfarmer** [comment]
Will it be the same time as your office hours?

**Comment by mikeknows** [comment]
> 
> 
> Will it be the same time as you office ours?

Yes and by appointment

**Pull request closed without merging**